test/cypress/integration/payment-links/delete_payment_links_spec.js

```diff
@@ -0,0 +1,52 @@
+const commonStubs = require('../../utils/common_stubs')
+const { getProductsStub, getProductByExternalIdStub, deleteProductStub } = require('../../utils/products_stubs')
+const userExternalId = 'a-user-id'
+const gatewayAccountId = 42
+const productExternalId = 'a-product-id'
+
+const buildPaymentLinkOpts = function buildPaymentLinkOpts (name, href, language) {
+  return {
+    external_id: productExternalId,
+    name: name,
+    language: language,
+    type: 'ADHOC',
+    links: [
+      {
+        'rel': 'friendly',
+        'method': 'GET',
+        'href': href
+      }
+    ]
+  }
```

Comment:
I think we don't have to provide any of these options apart from `external_id`, we can just use the defaults given to us by the fixture hopefully as `external_id` is the only thing used by any of our tests.

Reply:
I just removed it but it broke the test